lib/ansible/module_utils/network/iosxr/utils/utils.py

```diff
@@ -124,20 +145,22 @@ def _get_number(name):
         if_type = 'FastEthernet'
     elif name.lower().startswith('fo'):
         if_type = 'FortyGigE'
-    elif name.lower().startswith('et'):
-        if_type = 'Ethernet'
-    elif name.lower().startswith('vl'):
-        if_type = 'Vlan'
-    elif name.lower().startswith('lo'):
-        if_type = 'loopback'
-    elif name.lower().startswith('po'):
-        if_type = 'port-channel'
-    elif name.lower().startswith('nv'):
-        if_type = 'nve'
+    elif name.lower().startswith('te'):
+        if_type = 'TenGigE'
     elif name.lower().startswith('twe'):
         if_type = 'TwentyFiveGigE'
     elif name.lower().startswith('hu'):
         if_type = 'HundredGigE'
+    elif name.lower().startswith('vl'):
+        if_type = 'Vlan'
+    elif name.lower().startswith('lo'):
+        if_type = 'Loopback'
+    elif name.lower().startswith('be'):
+        if_type = 'Bundle-Ether'
+    elif name.lower().startswith('bp'):
+        if_type = 'Bundle-POS'
+    elif name.lower().startswith('mg'):
```

Comment:
It's better not to include mgmt interface here in normalize inteface as otherwise wrt to l2 and l3 interfaces in case of override and delete w/o config, even mgmt interface config will also get effected